docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>Before <code>cat_model.predict(idata, data=new_data, kind="mean")</code> is performed, the only data variable in the inference data posterior is <code>color_id</code>. However, after performing predictions on new data (or on the observed data), another data variable <code>censored(days_to_event / 31, adopt)</code> is populated. </p><p> </p><p><code>censored(days_to_event / 31, adopt)</code> is on the scale of $Y$ (in this example months). <code>color_id</code> is on the rate scale $lambda$. However, $lambda$ can also be calculated from <code>censored(days_to_event / 31, adopt)</code> by doing <code>1 / censored(days_to_event / 31, adopt) </code>. </p><p> </p><p>I am pointing this out because there is a "large" difference in the survival curves.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>First I didn't understand what you were trying to do. Then I got worried. Finally, I realized we were missing the link function!  Do what's below, and see it matches the posterior draws of <code>color_id</code>  <span class="ql-cursor">﻿</span></p><p> </p><pre class="ql-syntax" spellcheck="false">np.log(idata["posterior"]["censored(days_to_event / 31, adopt)_mean"])
</pre>

Reply:
Indeed, I was missing that!